dlworkspace-deployment/azuredeploy.json

```diff
@@ -4,11 +4,24 @@
     "parameters": {
         "clusterNameDesired": {
             "type": "string",
-            "defaultValue": "[resourceGroup().name]",
+            "defaultValue": "[uniqueString(resourceGroup().name)]",
```

Comment:
resourcgroup names are not globally unique, use:

[uniqueString(resourceGroup().id)]

Reply:
Done